db/migrations/3_extrinsic_hex_prefix.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE extrinsics
+SET extrinsic_hash = CONCAT('0x', extrinsic_hash)
+WHERE extrinsic_hash NOT LIKE '0x%';
```

Comment:
Does this PR require a resync so the old params get populated?

If so, then we don't need to update existing data.